canbench-bin/src/lib.rs

```diff
@@ -90,39 +91,41 @@ fn canbench_dir() -> PathBuf {
         .join(".canbench")
 }
 
-// Path to drun.
-fn drun_path() -> PathBuf {
-    canbench_dir().join("drun")
+// Path to PocketIC.
+fn pocket_ic_path() -> PathBuf {
+    canbench_dir().join("pocket-ic")
 }
 
-// Downloads drun if it's not already downloaded.
-fn maybe_download_drun(verbose: bool) {
-    const DRUN_LINUX_SHA: &str = "182b800a7979e1e3e516e54e4b9980e5407ced7464c0b3aec9ff7af6e9e69a1b";
-    const DRUN_MAC_SHA: &str = "8e0d0758d5a5c6f367e2c374dc7eae0106c7f46a3457f81018af6d5159d2dad4";
+// Downloads PocketIC if it's not already downloaded.
+fn maybe_download_pocket_ic(verbose: bool) {
+    const POCKET_IC_LINUX_SHA: &str =
+        "15feab44ebdc7b4b0157edd1a37ca07b8286262e7798abd531a5747b8fa8ea82";
+    const POCKET_IC_MAC_SHA: &str =
+        "fc536d3d582575df799ff3de0303ea383c68e5dd99754469303d55b6c32fd160";
 
-    if drun_path().exists() {
-        // Drun found. Verify that it's the version we expect it to be.
+    if pocket_ic_path().exists() {
+        // PocketIC found. Verify that it's the version we expect it to be.
         let expected_sha = match env::consts::OS {
-            "linux" => DRUN_LINUX_SHA,
-            "macos" => DRUN_MAC_SHA,
+            "linux" => POCKET_IC_LINUX_SHA,
+            "macos" => POCKET_IC_MAC_SHA,
             _ => panic!("only linux and macos are currently supported."),
         };
 
-        let drun_sha = sha256::try_digest(drun_path()).unwrap();
+        let pocket_ic_sha = sha256::try_digest(pocket_ic_path()).unwrap();
 
-        if drun_sha == expected_sha {
-            // Shas match. No need to download drun.
+        if pocket_ic_sha == expected_sha {
+            // Shas match. No need to download PocketIC.
             return;
         }
     }
 
-    // The expected version of drun isn't present. Download it.
-    download_drun(verbose);
+    // The expected version of PocketIC isn't present. Download it.
+    download_pocket_ic(verbose);
 }
 
-fn download_drun(verbose: bool) {
-    const DRUN_URL_PREFIX: &str =
-        "https://github.com/dfinity/ic/releases/download/release-2024-01-25_14-09/drun-x86_64-";
+fn download_pocket_ic(verbose: bool) {
+    const POCKET_IC_URL_PREFIX: &str =
+        "https://download.dfinity.systems/ic/5bcbdc7848400491ca9b738f3652a22094f21a3b/binaries/x86_64-";
```

Comment:
TODO: This path should be changed when the next version of PocketIC is released. 
Something like `https://github.com/dfinity/pocketic/releases/download/3.0.1/pocket-ic-x86_64-linux.gz`